tests/components/microsoft_face/test_init.py

```diff
@@ -134,15 +134,15 @@ async def test_setup_component_test_entities(
     """Set up component."""
     aioclient_mock.get(
         ENDPOINT_URL.format("persongroups"),
-        text=load_fixture("microsoft_face_persongroups.json"),
+        text=load_fixture("microsoft_face/persongroups.json"),
```

Comment:
This is preferred appraoch:
```suggestion
        text=load_fixture("persongroups.json", "microsoft_face"),
```